python/CMakeLists.txt

```diff
@@ -18,118 +23,185 @@ function(python_module)
     configure_file(
       "${MODULE_FILE}"
       "${CMAKE_BINARY_DIR}/${PYTHON_INSTALL_PATH}/${MODULE_FILE}" COPYONLY)
+    configure_file(
+      "${MODULE_FILE}"
+      "${CMAKE_BINARY_DIR}/build-wheel/${PYTHON_MODULE_WHEEL}/${MODULE_FILE}"
+      COPYONLY)
+    list(
+      APPEND "WHEEL_FILES-${PYTHON_MODULE_WHEEL}"
+      "${CMAKE_BINARY_DIR}/build-wheel/${PYTHON_MODULE_WHEEL}/${MODULE_FILE}")
   endforeach()
 
-  foreach(MODULE_FILE ${PYTHON_MODULE_MODULE_FILES}
-                      ${PYTHON_MODULE_MODULE_GENERATED_FILES})
-    # Copy module in the root at install time
-    get_filename_component(MODULE_FILE_DIR "${MODULE_FILE}" DIRECTORY)
-    install(
-      FILES "${CMAKE_BINARY_DIR}/${PYTHON_INSTALL_PATH}/${MODULE_FILE}"
-      DESTINATION
-        "${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_PATH}/${MODULE_FILE_DIR}")
+  foreach(MODULE_FILE ${PYTHON_MODULE_MODULE_GENERATED_FILES})
+    add_custom_command(
+      OUTPUT
+        "${CMAKE_BINARY_DIR}/build-wheel/${PYTHON_MODULE_WHEEL}/${MODULE_FILE}"
+      DEPENDS "${CMAKE_BINARY_DIR}/${PYTHON_INSTALL_PATH}/${MODULE_FILE}"
+      COMMAND
+        ${CMAKE_COMMAND} -E copy
```

Comment:
I would just use `cp`. We don't really support anything non-Linux for building as of now and we're unlikely to ever support building on Windows.

Reply:
Replaced with `cp`